sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -92,6 +95,24 @@ case class TransformWithStateExec(
     }
   }
 
+  private def getDriverProcessorHandle: DriverStatefulProcessorHandleImpl = {
+    val driverProcessorHandle = new DriverStatefulProcessorHandleImpl
+    statefulProcessor.setHandle(driverProcessorHandle)
+    statefulProcessor.init(outputMode, timeMode)
```

Comment:
Are we also adding a pre-init phase to track this ?

Reply:
Hm, I didn't really see the point of this, but we can

Reply:
Oh sure, will add it